docs/docs/Getting-Started/welcome-to-langflow.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Welcome to Langflow
+sidebar_position: 0
+slug: /
+---
+
+Langflow is a new, visual framework for building multi-agent and RAG applications. It is open-source, Python-powered, fully customizable, and LLM and vector store agnostic.
+
+Its intuitive interface allows for easy manipulation of AI building blocks, enabling developers to quickly prototype and turn their ideas into powerful, real-world solutions.
+
+Langflow empowers developers to rapidly prototype and build AI applications with its user-friendly interface and powerful features. Whether you're a seasoned AI developer or just starting out, Langflow provides the tools you need to bring your AI ideas to life.
+Get started with Langflow today and unlock the potential of AI for your projects!
+
+## Visual Flow Builder
```

Comment:
Use sentence case capitalization for headings (this comment applies to all headings in this PR).